odoo/service/server.py

```diff
@@ -159,6 +159,14 @@ def end_headers(self, *a, **kw):
             self.rfile = BytesIO()
             self.wfile = BytesIO()
 
+    @property
+    def path(self):
+        return self._opath + getattr(threading.current_thread(), 'rpc_path', '')
```

Comment:
This is incorrect and potentially breaks behaviour. I think it is not ok for just logging.

Can't you handle this using `odoo.netsvc.LogRecord`?